web/src/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -1,51 +1,47 @@
+import { ChecklistTag } from "@/components/ChecklistTag";
+import { HorizontalLine } from "@/components/HorizontalLine";
+import { Heading } from "@/components/njwds-extended/Heading";
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
 import { Icon } from "@/components/njwds/Icon";
+import { getAgencyText } from "@/components/TaskElement";
 import { getMergedConfig } from "@/contexts/configContext";
 import { useUserData } from "@/lib/data-hooks/useUserData";
 import analytics from "@/lib/utils/analytics";
 import { LicenseStatus, LicenseStatusItem } from "@businessnjgovnavigator/shared/";
-import { ReactElement, useEffect, useState } from "react";
+import { ReactElement } from "react";
 
 interface Props {
   items: LicenseStatusItem[];
   status: LicenseStatus;
+  agency: string;
+  context: string;
   onEdit: () => void;
 }
 
 type PermitTheme = {
-  gradient: string;
   bgColor: string;
+  bgSubHdrColor: string;
   textColor: string;
-  borderColor: string;
-  iconColor: string;
-  headerIconColor: string;
 };
 
 const Config = getMergedConfig();
-const pendingPermitTheme: PermitTheme = {
-  gradient: "gradient-blue",
-  bgColor: "bg-info-extra-light",
-  textColor: "text-cyan",
-  borderColor: "border-cyan",
-  iconColor: "text-base-lighter",
-  headerIconColor: "text-cyan",
-};
 
 const activePermitTheme: PermitTheme = {
```

Comment:
pebble - the variable name should be updated for all three of these variables, because they no longer describe the theme, they only describe the permit status component